package.json

```diff
@@ -11,7 +11,7 @@
     "build:watch": "tsc -p tsconfig.json -watch",
     "clean": "git checkout HEAD -- sandbox && git clean -f -d sandbox",
     "build:clean:launch": "yarn build && yarn clean && yarn launch",
-    "launch": "cd sandbox && ng g @briebug/jest-schematic:jest",
+    "launch": "yarn link:sandbox && cd sandbox && yarn && ./node_modules/.bin/ng g @briebug/jest-schematic:jest && yarn",
```

Comment:
thanks for this PR. The last `&& yarn` from the `launch` command seems redundant? I think it can be removed.

Reply:
@schuchard Thanks for the heads up. You totally right. Code is updated. :)